airgun/entities/dashboard.py

```diff
@@ -0,0 +1,42 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.dashboard import DashboardView
+
+
+class DashboardEntity(BaseEntity):
+
+    def search(self, value):
+        """Initiate search procedure that applied on all dashboard widgets.
+        Return widgets values as a result
+        """
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def read(self):
```

Comment:
`read` sounds more generic as anyway we can't have any other alternative way to read something

it takes around 8-10 seconds. I thought about that idea, but not sure if that has real impact as it is not very generic case. It is like saving around 1-2 minutes for whole test suite in one thread, but add more logic that required from test level.
Let's me know your decision as it will require re-implementing all cases

Reply:
okay, it takes more than 10 seconds :(